### PR TITLE
refactor(aws_root_disk_name): info is found in the

### PR DESCRIPTION
**Pull request by fgelcer**
AMI parameters, hence removing this entry from all
sct_config. It applies for DB, loader and monitor
nodes.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [x] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fgelcer** [comment]
ping @fruch

File: sdcm/tester.py [hunk 1]
```diff
@@ -568,6 +568,7 @@ def get_cluster_gce(self, loader_info, db_info, monitor_info):
 
     def get_cluster_aws(self, loader_info, db_info, monitor_info):
         # pylint: disable=too-many-locals,too-many-statements,too-many-branches
+        ec2 = boto3.resource('ec2')
```
Comment (bentsi) [inline sdcm/tester.py:571]: no `region_name` parameter specified, this won't work for nondefault regions

**Comment by bentsi** [comment]
this is a refactor, not a fix for a bug, hence no need to backport

File: sdcm/tester.py (hunk 1 above)
Comment (fgelcer) [reply sdcm/tester.py]: Fixed.
Testing the new solution.

**Comment by fgelcer** [comment]
> this is a refactor, not a fix for a bug, hence no need to backport

in case we will want to run amazon linux 2 on 4.1, 4.0 or even 2020, we will need this code, as it is a refactor to fix an issue...

File: sdcm/tester.py (hunk 1 above)
Comment (fgelcer) [reply sdcm/tester.py]: tests have passed, ready to merge

**Comment by fgelcer** [comment]
> Have you considered the option of moving the code of device mapping creation closer to the instance creation? In this case you just need to pass disk size parameter.
> 
> Also there is unused constant `sdcm.cluster.SCYLLA_CLUSTER_DEVICE_MAPPINGS` which can be removed in this PR too, I think.

i thought about it on later stage, but, for some reason some of the data i need to manipulate (the `dict()`) was much harder to change, so i decided to enter the data all together.
i removed `sdcm.cluster.SCYLLA_CLUSTER_DEVICE_MAPPINGS` and i'm checking if everything works as before removing it.

**Comment by fgelcer** [comment]
@enaydanov , after removing the `sdcm.cluster.SCYLLA_CLUSTER_DEVICE_MAPPINGS` a job seems to be working normally, hence this PR is ready to be merged.

**Pull request merged**